server/public/shared/mlog/mlog.go

```diff
@@ -98,32 +98,25 @@ func (lc LoggerConfiguration) toTargetCfg() map[string]logrcfg.TargetCfg {
 // will be used to generate a string representation.
 var Any = logr.Any
 
-// Int64 constructs a field containing a key and Int64 value.
-var Int64 = logr.Int64
```

Comment:
This is a breaking change in a `public` package.  Any apps importing mlog will break.  I think we can deprecate the  specific typed methods but deleting them will require a full version bump.

Reply:
`github.com/mattermost/mattermost-server/server/public` is not yet on v1. In https://docs.mattermost.com/upgrade/important-upgrade-notes.html, we stated 

> While this submodule has not yet shipped a v1 and will introduce breaking changes before stabilizing the API, it remains both forwards and backwards compatible with the Mattermost server itself.

One of the main points of the submodule was to allow breaking changes for now.